pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
@hughsaunders But https://github.com/rcbops/rpc-gating/blob/master/playbooks/setup_jenkins_slave.yml#L126 is installing an unknown version of setuptools on the slave managing the setup of the new slave.